systemtest/src/main/java/io/strimzi/systemtest/utils/FileUtils.java

```diff
@@ -30,132 +30,73 @@ private FileUtils() { }
 
     @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE")
     public static File downloadAndUnzip(String url) {
-        File dir = null;
-        FileOutputStream fout = null;
-        ZipInputStream zin = null;
-        try {
-            InputStream bais = (InputStream) URI.create(url).toURL().openConnection().getContent();
-            dir = Files.createTempDirectory(FileUtils.class.getName()).toFile();
+
+        try (InputStream bais = (InputStream) URI.create(url).toURL().openConnection().getContent();
+             ZipInputStream zin = new ZipInputStream(bais)){
+
+            File dir = Files.createTempDirectory(FileUtils.class.getName()).toFile();
             dir.deleteOnExit();
-            zin = new ZipInputStream(bais);
+
             ZipEntry entry = zin.getNextEntry();
             byte[] buffer = new byte[8 * 1024];
             int len;
             while (entry != null) {
                 File file = new File(dir, entry.getName());
-                if (entry.isDirectory()) {
-                    file.mkdirs();
-                } else {
-                    fout = new FileOutputStream(file);
+                try (FileOutputStream fout = new FileOutputStream(file)) {
                     while ((len = zin.read(buffer)) != -1) {
                         fout.write(buffer, 0, len);
                     }
-                    fout.close();
                 }
                 entry = zin.getNextEntry();
             }
         } catch (IOException e) {
             LOGGER.error("IOException {}", e.getMessage());
-        } finally {
-            if (fout != null) {
-                try {
-                    fout.close();
-                } catch (IOException e) {
-                    LOGGER.error("IOException {}", e.getMessage());
-                }
-            }
-            if (zin != null) {
-                try {
-                    zin.close();
-                } catch (IOException e) {
-                    e.printStackTrace();
-                }
-            }
         }
-        return dir;
+        return null;
     }
 
-    public static File downloadYamlAndReplaceNameSpace(String url, String namespace) {
-        InputStream bais = null;
-        BufferedReader br = null;
-        OutputStreamWriter osw = null;
+    public static File downloadYamlAndReplaceNameSpace(String url, String namespace) throws IOException {
```

Comment:
I guess we should fix the captialization `NameSpace` → `Namespace`

Reply:
make sense )